.github/workflows/reusable-run-checker.yml

```diff
@@ -13,11 +13,14 @@ on:
         description: If lint for protos need to run need to be run, will be 'true'
         value: ${{ jobs.changes.outputs.lint_workflow == 'true' || jobs.changes.outputs.proto == 'true' }}
       run_lint_rust:
-        description: If lint for rust need to run need to be run, will be 'true'
+        description: If lint for rust needs to run need to be run, will be 'true'
```

Comment:
```suggestion
        description: If lint for rust needs to be run, will be 'true'
```

Reply:
Looks like this was from a copy/paste from the previous `run_lint_proto` and is copied a couple of times below as well.